chunky/src/res/se/llbit/chunky/ui/Credits.fxml

```diff
@@ -8,81 +8,141 @@
+                                 <Insets left="10.0" />
+                              </padding>
+                           </VBox>
+                        </children>
+                     </VBox>
+                     <VBox spacing="10.0">
+                        <children>
+                           <Label text="Lead Developers">
+                              <font>
+                                 <Font name="System Bold" size="12.0" />
+                              </font>
+                           </Label>
+                           <VBox>
+                              <children>
+                                 <Label text="Jesper Öqvist" />
+                                 <Label text="leMaik" />
+                              </children>
+                              <padding>
+                                 <Insets left="10.0" />
+                              </padding>
+                           </VBox>
+                           <Label text="Contributors">
+                              <font>
+                                 <Font name="System Bold" size="12.0" />
+                              </font>
+                           </Label>
+                           <VBox>
+                              <children>
+                                 <Label text="a_Tom" />
+                                 <Label text="TOGoS" />
+                                 <Label text="Redox (alexhliu)" />
+                                 <Label text="jackjt8" />
+                                 <Label text="073TheBoss" />
+                                 <Label text="NotStirred" />
+                                 <Label text="Ice Selkie (StanleyMines)" />
+
+                                 <Hyperlink fx:id="ghContributors" text="Full List" />
+                              </children>
+                              <padding>
+                                 <Insets left="10.0" />
+                              </padding>
+                           </VBox>
+                        </children>
+                     </VBox>
+                  </children>
+                  <padding>
+                     <Insets left="10.0" right="10.0" top="10.0" />
+                  </padding>
+               </HBox>
+
+               <HBox spacing="10.0">
+                  <children>
+                     <VBox spacing="10.0">
+                        <padding>
+                           <Insets bottom="10.0" left="10.0" right="10.0"/>
+                        </padding>
+                        <children>
+                           <Label text="Tools and Libraries">
+                              <font>
+                                 <Font name="System Bold" size="12.0" />
+                              </font>
+                           </Label>
+                           <VBox>
+                              <children>
+                                 <Hyperlink fx:id="markdown" text="Markdown">
```

Comment:
Where does chunky use that btw.? Would be cool to have markdown for the changelogs in the launcher!